flatm8s/src/App.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import SignUp from './SignUp'
 import Home from './Home'
 import Error from './Error'
+import Login from "./Login"
```

Comment:
@lse30 , here you have imported login but you haven't checked in the login.js file so it fails to compile